lib/validate.js

```diff
@@ -8,7 +8,7 @@ const _ = require('lodash');
 
 function getEntryForFunction(serverlessFunction) {
 	const handler = serverlessFunction.handler;
-	const handlerFile = /(.*)\..*?$/.exec(handler)[1] + '.js';
+	const handlerFile = /(.*)\..*?$/.exec(handler)[1];
```

Comment:
Better rename the `handlerFileBaseName` variable to `handlerFilePathWithBaseName` as it actually contains the path + the basename.